final_contact_threshold: 0.05
chassis_contact_threshold: 0.1
convexity_threshold: 0.2
